src/PKSim.Core/Services/PKAnalysesTask.cs

```diff
@@ -40,6 +41,10 @@ public interface IPKAnalysesTask : OSPSuite.Core.Domain.Services.IPKAnalysesTask
 
       IndividualPKAnalysis CalculateFor(Simulation simulation, DataColumn dataColumn);
       PKValues CalculatePK(DataColumn column, PKCalculationOptions options);
+
+      PKAnalysis CreatePKAnalysisFromValues(PKValues pkValues, Simulation simulation, Compound compound);
+      string LowerSuffix(string text);
```

Comment:
Add a comment to this interface (and the one below) explaining what this methods do. Also rename parameter text to something meaningful

Reply:
> does it need to be public at all?

It is being used outside the class.

Reply:
> > does it need to be public at all?
> 
> It's being used outside the class.

Not anymore... it is now private ;)